plasmapy/physics/quantum.py

```diff
@@ -99,3 +103,187 @@ def deBroglie_wavelength(V, particle):
             lambda_dBr = h / (Lorentz_factor(V) * m * V)
 
     return lambda_dBr.to(units.m)
+
+
+@check_quantity({
+    'T_e': {'units': units.K, 'can_be_negative': False}
+})
+def thermal_deBroglie_wavelength(T_e):
+    r"""Calculate the thermal deBroglie wavelength for electrons.
+
+    Parameters
+    ----------
+    T_e: Quantity
+        Electron temperature
+
+    Returns
+    -------
+    lambda_dbTh: Quantity
+        The thermal deBroglie wavelength for electrons in meters
+
+    Raises
+    ------
+    TypeError
+        If argument is not a Quantity
+
+    UnitConversionError
+        If argument is in incorrect units
+
+    ValueError
+        If argument contains invalid values
+
+    UserWarning
+        If units are not provided and SI units are assumed
+
+    Notes
+    -----
+    The thermal deBroglie wavelength is approximately the average deBroglie
+    wavelength for electrons in an ideal gas and is given by
+
+    .. math::
+    \lambda_dbTh = \frac{h}{\sqrt{2 \pi m_e k_B T_e}}
+
+    See also
+    --------
+    
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> thermal_deBroglie_wavelength(1 * u.eV)
+    <Quantity 6.919367518364532e-10 m>
+
+    """
+    T_e = T_e.to(units.K, equivalencies=units.temperature_energy())
+    try:
+        lambda_dbTh = h / np.sqrt(2 * np.pi * m_e * k_B * T_e)
+    except Exception:
+        raise ValueError("Unable to find thermal deBroglie wavelength.")
+    return lambda_dbTh.to(units.m)
+
+@check_quantity({
+    'n_e': {'units': units.m**-3, 'can_be_negative': False}
+})
+def Fermi_energy(n_e):
+    r"""Calculate the Thomas-Fermi screening length.
```

Comment:
Should this be `"Calculate the Fermi energy"` instead?

Reply:
good catch!